_content/the-things-stack-v3/migrate-to-v3.md

```diff
@@ -0,0 +1,4 @@
+---
+title: How do I migrate to The Things Stack V3?
+---
```

Comment:
This this be filled in?

Reply:
yes, I'm still working on the migration part and FAQ

Reply:
@johanstokking Can you please spend some time to review this? I'm still left with FAQ, but that can be updated along the way with new questions. This was the idea:
- info page - provide info about what V3 is, where to find it, why to migrate
- migration page - unite the overall migration process in a single place on ttn.org/docs, link the relevant documentation so people know where to find more details